src/main/java/uk/gov/hmcts/dts/fact/services/admin/AdminCourtTypesService.java

```diff
@@ -0,0 +1,129 @@
+package uk.gov.hmcts.dts.fact.services.admin;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+import uk.gov.hmcts.dts.fact.entity.Court;
+import uk.gov.hmcts.dts.fact.exception.NotFoundException;
+import uk.gov.hmcts.dts.fact.model.admin.CourtType;
+import uk.gov.hmcts.dts.fact.repositories.CourtRepository;
+import uk.gov.hmcts.dts.fact.repositories.CourtTypeRepository;
+
+import java.util.List;
+
+import static java.util.stream.Collectors.toList;
+
+@Service
+public class AdminCourtTypesService {
+    private final CourtRepository courtRepository;
+    private final CourtTypeRepository courtTypeRepository;
+
+    @Autowired
+    public AdminCourtTypesService(final CourtRepository courtRepository, final CourtTypeRepository courtTypeRepository) {
+        this.courtRepository = courtRepository;
+        this.courtTypeRepository = courtTypeRepository;
+    }
+
+
+    public List<CourtType> getAllCourtTypes() {
+        return  courtTypeRepository.findAll()
+            .stream()
+            .map(CourtType::new)
+            .collect(toList());
+
+
```

Comment:
Redundant empty spaces in this method and some other methods in this class

Reply:
Removed empty spaces